src/app/components/Search/index.tsx

```diff
@@ -105,6 +107,7 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
   const [value, setValue] = useState('')
   const [isFocused, setIsFocused] = useState(false)
   const valueInSearchParams = useSearchParams()[0].get('q') ?? ''
+  const [itsTimeToAct, setItsTimeToAct] = useState(false)
```

Comment:
I dont think this variable name is clear

Reply:
Perhaps a better structure is: a delayed "has user stopped typing", instead of delayed "has a problem"

Reply:
OK, as suggested, now we have `isTyping`, which is always set to true when typing, and then set to false in a delayed manner. (And we display messages when !isTyping.)